docs/docs/how-to-install-packages.md

```diff
@@ -0,0 +1,21 @@
+---
+title: How to install packages
+slug: /how-to-install-packages
+tags:
+  - lerna
+  - packages
+---
+
+# Best practices for installing packages in Strapi
+
+When working with the Strapi mono repo, it's important to follow best practices for installing packages to avoid potential issues and ensure consistent results. Instead of using the standard **`yarn add`** command, we recommend using **`yarn lerna add <package_name> --scope @strapi/<module_name>`** for installing packages.
```

Comment:
```suggestion
When working with the Strapi monorepo, it's important to follow best practices for installing packages to avoid potential issues and ensure consistent results. Instead of using the standard **`yarn add`** command, we recommend using **`yarn lerna add <package_name> --scope @strapi/<module_name>`** for installing packages.
```

i believe monorepo is one word :)